### DISCUSSION TITLE
Remote website monitoring with NSCP

### DISCUSSION THREAD

**Discussion by jbeaujour** [mickem/nscp#810] [Q&A]
Hello
Thanks for nscp !!
We are looking testing a website url with NSCP.
The purpose is that NAGIOS service will use check_nrpe (or another NSCP client) toward NSCP and execute a plugin like check_http to monitor a website on Internet (eg wikipedia)
We don't want to use check_by_ssh or external_scripts in NSCP
Thank's for help

**Comment by mickem** [comment]
There is now a built in check_http command for this
